storage/client/src/main/java/com/azure/storage/blob/BlockBlobAsyncClient.java

```diff
@@ -184,9 +230,9 @@ public static BlockBlobClientBuilder builder() {
      * @return
      *      A reactive response signalling completion.
      */
-    public Mono<Void> stageBlock(String base64BlockID, Flux<ByteBuf> data,
+    public Mono<Response<BlockBlobItem>> stageBlock(String base64BlockID, Flux<ByteBuf> data,
```

Comment:
I was under the impression that only the Raw clients (or the Raw methods) accept Flux<ByteBuf> and the ones on the "simple" clients would accept/return Flux<ByteBuffer>

Reply:
Fixed.